.storybook/preview.js

```diff
@@ -1,9 +1,9 @@
+import '@fortawesome/fontawesome-free/css/all.css';
+import '@fortawesome/fontawesome-free/css/v4-shims.css';
```

Comment:
we're using those in the backend project and I'd like to keep the icons the same for now